### PR TITLE
Nowe pliki:

### PR DESCRIPTION
**Pull request by dkleszczynski**
- Pet.cs - plik modelu
- PetSex - plec zwierzaka
- PetType - typ zwierzaka (pies czy kot)

### CODE REVIEW

**Comment by lsielewicz** [comment]
In c# there is a little bit different naming convention in case of enums. Only first letter should be capital, rest should be normal. Please change it to the proper way :)

File: Models/EnumTypes/PetSex.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace MvcApp.Models.EnumTypes
```
Comment (lsielewicz) [inline Models/EnumTypes/PetSex.cs]: I see, that there is also wrong namespace used. Please fix

Comment (dkleszczynski) [inline Models/EnumTypes/PetSex.cs]: Do you mean I should use `MvcApp.Models` namespace instead?

Comment (lsielewicz) [inline Models/EnumTypes/PetSex.cs]: I think nameapace should point to the real localization, so in this case should be TOZ_dotnet.Models. Anyway I would move our models to TOZ_dotnet.Core project but it's a case to discuss with the other members.

Comment (htaler) [inline Models/EnumTypes/PetSex.cs]: I agree with @lsielewicz but please dont use underscore (_) in namespaces!
Toz.Models would be enough

**Comment by htaler** [comment]
{get; private set;} - please modify as it can cause issues during deserialization

**Comment by lsielewicz** [comment]
There is already a newer version of the same pull request (duplicated), so I'm closing this one.

**Pull request closed without merging**